bin/ci/after-wp-install.sh

```diff
@@ -39,7 +39,7 @@ fi
 if [[ -n $INSTALL_PWA_PLUGIN ]]; then
 	echo -n "Installing PWA plugin..."
 	wget -O "$WP_CORE_DIR/src/wp-content/plugins/pwa.zip" https://downloads.wordpress.org/plugin/pwa.zip
-	unzip -d "$WP_CORE_DIR/src/wp-content/plugins/pwa/" "$WP_CORE_DIR/src/wp-content/plugins/pwa.zip"
```

Comment:
How did you come across this?

Reply:
Noticed a series of skipped tests in the PHPUnit output, and traced it back to the `Test_AMP_Service_Worker` class being skipped. Those tests are only skipped when the PWA plugin is not active.

Reply:
The PWA plugin was being extracted to the directory `$WP_CORE_DIR/src/wp-content/plugins/pwa/pwa`, which didn't match the hardcoded path used to activate the plugin.